pkg/accounts/adaptor/controller/account.ts

```diff
@@ -271,12 +271,12 @@ export class AccountController {
     const headerRes = await this.headerService.fetchByAccountID(
       account.getID(),
     );
-    const avatar = Result.isOk(avatarRes)
-      ? Result.unwrap(avatarRes).getUrl()
-      : '';
-    const header = Result.isOk(headerRes)
-      ? Result.unwrap(headerRes).getUrl()
-      : '';
+    const avatar = Result.mapOr('')((avatarImage: Medium): string =>
+      avatarImage.getUrl(),
+    )(avatarRes);
+    const header = Result.mapOr('')((headerImage: Medium): string =>
+      headerImage.getUrl(),
+    )(headerRes);
```

Comment:
```suggestion
    const avatar = Result.mapOr('')(getMediumUrl)(avatarRes);
    const header = Result.mapOr('')(getMediumUrl)(headerRes);
```
任意ですが, `getUrl` を呼び出す `getMediumUrl` 関数を `Medium` の定義ファイルにでも用意しておくと, 少し簡潔に書けると思います.

Reply:
これは別PRで対応しようと思います